test/parallel/test-tls-handshake-error.js

```diff
@@ -9,8 +10,8 @@ const tls = require('tls');
 const fs = require('fs');
 
 const server = tls.createServer({
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`),
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
+  cert: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-cert.pem`),
```

Comment:
This should use the `fixtures.readKey` function instead.

Reply:
For instance,

```js
key: fixtures.readKey('agent1-key.pem'),
cert: fixtures.readKey('agent1-cert.pem')
```